### DISCUSSION TITLE
Master no longer passing tests

### DISCUSSION THREAD

**Discussion by lachlanbelcher** [nwchemgit/nwchem#772] [Q&A] [answered]
Hi,  my master branch is up to date with Nwchem:master, but it's failing all the tests.   If it's an exact copy of the nwchem master, how can it fail? 

Thanks

**Comment by jeffhammond** [comment]
Bad compiler. Bad MPI. Etc. Please share an output file. It will help us help you.

**Reply by jeffhammond** [reply]
It's also possible somebody pushed a bug but that's relatively rare given the nice CI that Edo built.

**Reply by lachlanbelcher** [reply]
https://github.com/lachlanbelcher/nwchem/actions/runs/4746154310

Here is a link to the workflow actions.  Are you able to access that?

**Reply by dmejiar** [reply]
@lachlanbelcher there seems to be an issue with the cache of GitHub actions. For some reason, the CI workflow is assigning an Xcode version to an Ubuntu build, but that cache does not exist:
```
2023-04-19T17:25:10.1086958Z with:
2023-04-19T17:25:10.1087163Z   path: ~/cache

2023-04-19T17:25:10.1087597Z   key: ubuntu-20.04-mpich-gfortran-xcode13.2.1-simint-blasbuild_openblas-blas_size8-libxc-tblite-simdavx512-nwchemcache-v003
2023-04-19T17:25:10.1088055Z   enableCrossOsArchive: false
2023-04-19T17:25:10.1088306Z   fail-on-cache-miss: false
2023-04-19T17:25:10.1088545Z   lookup-only: false
```

**Reply by dmejiar** [reply]
That was my mistake, the cache does exist. Still, there seems to be an issue with mpich coming out from the cache file

**Comment by edoapra** [comment, marked as answer]
@lachlanbelcher you need to clear your caches
Go to https://github.com/lachlanbelcher/nwchem/actions/workflows/clear_cache.yml
and click the "Run workflow" tab to the  left of "This workflow has a workflow_dispatch event trigger."

**Reply by lachlanbelcher** [reply]
Thanks, this helped.  On to the next issue!

**Comment by lachlanbelcher** [comment]
Hi I followed the link but I couldn't find that tab.

**Reply by edoapra** [reply]
> Hi I followed the link but I couldn't find that tab.
> […](#)
> On Thu, Apr 20, 2023 at 12:59 AM Edoardo Aprà ***@***.***> wrote: @lachlanbelcher <https://github.com/lachlanbelcher> you need to clear your caches Go to https://github.com/lachlanbelcher/nwchem/actions/workflows/clear_cache.yml and click the "Run workflow" tab to the left of "This workflow has a workflow_dispatch event trigger." — Reply to this email directly, view it on GitHub <[#772 (comment)](https://github.com/nwchemgit/nwchem/discussions/772#discussioncomment-5668722)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQX35RXO4XYRG43LDGIWYULXCCYBNANCNFSM6AAAAAAXEMXZUA> . You are receiving this because you were mentioned.Message ID: ***@***.***>

What link?
